sdk/contentsafety/ai-content-safety-rest/README.md

```diff
@@ -0,0 +1,465 @@
+
+| Category  | Description                                                                                                                                                                                                                                                                                                                                                                                         |
+| --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Hate      | Hate refers to any content that attacks or uses pejorative or discriminatory language in reference to a person or identity group based on certain differentiating attributes of that group. This includes but is not limited to race, ethnicity, nationality, gender identity and expression, sexual orientation, religion, immigration status, ability status, personal appearance, and body size. |
+| Sexual    | Sexual describes content related to anatomical organs and genitals, romantic relationships, acts portrayed in erotic or affectionate terms, pregnancy, physical sexual acts—including those acts portrayed as an assault or a forced sexual violent act against one’s will—, prostitution, pornography, and abuse.                                                                                  |
+| Violence  | Violence describes content related to physical actions intended to hurt, injure, damage, or kill someone or something. It also includes weapons, guns and related entities, such as manufacturers, associations, legislation, and similar.                                                                                                                                                          |
+| Self-harm | Self-harm describes content related to physical actions intended to purposely hurt, injure, or damage one’s body or kill oneself.                                                                                                                                                                                                                                                                   |
+
+Classification can be multi-labeled. For example, when a text sample goes through the text moderation model, it could be classified as both Sexual content and Violence.
+
+### Severity levels
+Every harm category the service applies also comes with a severity level rating. The severity level is meant to indicate the severity of the consequences of showing the flagged content.
+
+| Severity | Label  |
+| -------- | ------ |
+| 0        | Safe   |
+| 2        | Low    |
+| 4        | Medium |
+| 6        | High   |
```

Comment:
should we add undefined here ?

Reply:
I don't think we should add undefined here. The ACS API does not give undefined in its response body.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/26542#discussion_r1325834567 Could you explain this ?